e2e/browser/solid-ui-react/test/solidui.playwright.ts

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright Inrupt Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal in
+// the Software without restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+// Software, and to permit persons to whom the Software is furnished to do so,
+// subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+// PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+// SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+import { custom } from "openid-client";
+import { test, expect } from "./fixtures";
+
+custom.setHttpOptionsDefaults({
+  timeout: 15000,
+});
+
+test("solid-ui-react smoke test", async ({ app, testResource, auth }) => {
+  await app.home();
+  await expect(app.sampleRequest()).resolves.toEqual("Sample response");
+
+  await auth.login();
+  await expect(app.getWebid()).resolves.not.toBeNull();
+  await expect(app.fetchResource(testResource.url)).resolves.toEqual(
+    testResource.expectedContent,
+  );
```

Comment:
Can we also run this before login so we are testing the unauthenticated fetch for solid-ui-react.